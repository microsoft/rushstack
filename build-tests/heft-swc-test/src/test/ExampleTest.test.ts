// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
// See LICENSE in the project root for license information.

interface IInterface {
  element: string;
}

describe('Example Test', () => {
  it('Correctly tests stuff', () => {
    expect(true).toBeTruthy();
  });

  it('Correctly handles snapshots', () => {
    expect({ a: 1, b: 2, c: 3 }).toMatchSnapshot();
  });

  it('Correctly handles TypeScript constructs', () => {
    const interfaceInstance: IInterface = {
      element: 'a'
    };
    expect(interfaceInstance).toBeTruthy();
  });
});
