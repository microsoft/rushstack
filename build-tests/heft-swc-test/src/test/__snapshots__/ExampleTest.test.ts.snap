// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example Test Correctly handles snapshots 1`] = `
Object {
  "a": 1,
  "b": 2,
  "c": 3,
}
`;
